Added upgrade guide fix for HMR (#6446)
plone/volto: docs/source/upgrade-guide/index.md

```{versionadded} @plone/registry 3.0.0-alpha.0
```

### Add missing overrides to projects in `package.json`

This will fix some issues with Hot Module Reload in projects.

> It's required from Volto `18.0.0-alpha.47` otherwise, the site breaks in development mode.

Reviewer: ```suggestion
It's required in Volto `18.0.0-alpha.47` and later, otherwise the site breaks in development mode.
```

> Add this object to the `pnpm` key in your project `package.json`:

Reviewer: ```suggestion
Add this object to the `pnpm` key in your project {file}`package.json`.
```

> ```

Reviewer: ````suggestion
```json
````
